jetson/face_extractor.py

```diff
@@ -65,18 +40,17 @@ def save_cropped_img(frame, file_num):
 
 if __name__ == "__main__":
```

Comment:
why is a main method in this file? this shouldn't be here with other functionality. I think one of the 2 needs to leave. I personally would not have guessed this is here and I doubt others would have too. In general the main method should be in a clear location like `main.py`, `__main__.py` or `embedded2.py`